content/source/docs/extend/plugin-sdk.html.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: "extend"
+page_title: "Terraform Plugin SDK"
+sidebar_current: "docs-extend-0.12-compatibility"
+description: |-
+  Official standalone SDK for Terraform plugin development
+---
+
+As of September 2019, Terraform provider developers importing the Go module `github.com/hashicorp/terraform`, known as Terraform Core, should switch to `github.com/hashicorp/terraform-plugin-sdk`, the Terraform Plugin SDK, instead. 
```

Comment:
Prior to public release we should update this if necessary. We could also add the relevant Core version.